rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -34,12 +34,8 @@ License:    GPLv3
 URL:        http://github.com/theforeman/foreman_discovery
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}%{?prever}.gem
 
-Requires:   foreman >= 1.6.0
-Requires:   %{?scl_prefix}rubygem(open4)
-Requires:   %{?scl_prefix}rubygem(ftools)
-Requires:   advancecomp
-Requires:   squashfs-tools
-Requires:   sudo
+Requires:   foreman >= 1.7.0
```

Comment:
This is a bug, we require 1.7+. Filed a PR for RC2: https://github.com/theforeman/foreman_discovery/pull/128